Dockerfile

```diff
@@ -46,6 +46,13 @@ RUN curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontaine
 RUN apt update
 RUN apt -yq install podman
 
+# Upgrade npm
+RUN npm config set prefix /usr
+RUN npm install -g --force npm
+
+# Install Yarn
+RUN npm install -g yarn
```

Comment:
Are you sure this line installs yarn 2 and not the legacy yarn?

Reply:
Yes, but also no. It installs Yarn 1 globally. Yarn 2 is only installed at the project's directory, and Yarn 1 is used to call it. It's impossible to install Yarn 2 globally.
You can find Yarn 2 files for the CLI's tests in this PR:
https://github.com/jfrog/jfrog-cli/pull/1086/files#diff-235b6c62fe0fbe669bdc45165800e5a270addf2251e167fc1c8df524366a2299

Reply:
Got it @asafgabai. NP.